packages/components/src/lib/MarkdownViewer.tsx

```diff
@@ -120,13 +124,15 @@ export const MarkdownViewer: React.FC<MarkdownViewerProps> = ({
                   <SquigglePlayground
                     defaultCode={String(children).replace(/\n$/, "")}
                     height={height}
+                    linker={linker}
                   />
                 </div>
               );
             } else if (match && match[1] === "squiggleEditor") {
               return (
                 <div className="pb-4 pt-2">
                   <SquiggleEditor
+                    project={linker ? new SqProject({ linker }) : undefined}
```

Comment:
code previews in markdown logs were broken

Reply:
yea, I just didn't get to this. This seems good though!